src/ConfigManager.cpp

```diff
@@ -434,6 +434,14 @@ void ConfigManager::createBaseWebServer() {
 void ConfigManager::streamFile(const char* file, const char mime[]) {
   SPIFFS.begin();
 
+  if (file[0] != '/') {
+    size_t len = strlen(file);
+    char* path = new char[len+1];
+    strcpy (path, "/");
+    strcat (path, file);
+    file = path;
+  }
```

Comment:
uh .... you sir are a master of the pointer .. Lets see if Travis complains about the variable length .. 
Does this need a `delete(path)` for a clean-up ?

Reply:
yeah, but after `file` has been used. Cause file is now pointing at path. Nice catch.

Reply:
wait .. if file is pointing at path, and then we remove path (after we use file) -- is file now doinked next time we use it? maybe we need to use path like 

```
size_t len = strlen(file);
  char* path = new char[len+1];

  if (file[0] != '/') {
    strcpy (path, "/");
    strcat (path, file);
  } else {
   strcat (path, file);
}


File f = SPIFFS.open(file, "r");
```

Reply:
```
void ConfigManager::streamFile(const char* file, const char mime[]) {
  SPIFFS.begin();

  size_t len = strlen(file);
  char* path;

  if (file[0] != '/') {
    path = new char[len+1];
    strcpy (path, "/");
    strcat (path, file);
    file = path;
  } else {
    path = new char[len];
    strcat (path, file);
  }

  File f = SPIFFS.open(path, "r");
  if (f) {
    server->streamFile(f, FPSTR(mime));
    f.close();
  } else {
    DebugPrint(F("file open failed "));
    DebugPrintln(path);
    handleNotFound();
  }

  delete(path);
}
```